rplugin/python3/deoplete/sources/deoplete_jedi/server.py

```diff
@@ -395,7 +395,10 @@ def completion_dict(self, name, type_, comp):
                         continue
                     if '\\n' in desc:
                         desc = desc.replace('\\n', '\\x0A')
-                    params.append(desc)
+                    if desc.startswith('param'):
+                        desc = desc[5:].strip()
```

Comment:
Shouldn't that check for `'param '` (with space at the end), and then remove 6 chars?